src/pkg/repository/repository_unexported_test.go

```diff
@@ -51,3 +109,186 @@ func (suite *RepositoryModelSuite) TestWriteGetModel() {
 	require.NoError(t, err)
 	assert.Equal(t, "fnords", string(got.ID))
 }
+
+// helper func for writing backups
+//
+//revive:disable:context-as-argument
+func writeBackup(
+	t *testing.T,
+	ctx context.Context,
+	kw *kopia.Wrapper,
+	sw *store.Wrapper,
+	tID, snapID, backupID string,
+	sel selectors.Selector,
+	deets *details.Details,
+	errors fault.Errors,
+	errs *fault.Bus,
+) *backup.Backup {
+	//revive:enable:context-as-argument
```

Comment:
I found a less footgun-y way :P
```suggestion
func writeBackup(
	t *testing.T,
	ctx context.Context, //revive:disable-line:context-as-argument
	kw *kopia.Wrapper,
	sw *store.Wrapper,
	tID, snapID, backupID string,
	sel selectors.Selector,
	deets *details.Details,
	errors fault.Errors,
	errs *fault.Bus,
) *backup.Backup {
```

Reply:
YASS! 🎉